shell/platform/glfw/external_texture_gl.cc

```diff
@@ -0,0 +1,90 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/glfw/external_texture_gl.h"
+
+#include <memory.h>
+
+#include <glad/glad.h>
+
+// glad.h must be included before glfw3.h.
+#include <GLFW/glfw3.h>
+
+namespace flutter {
+
+// Just to not declare GLFWwindow and GLuint in the header.
+struct ExternalTextureGLState {
+  GLFWwindow* window;
+  GLuint gl_texture;
+};
+
+static void OnGLBufferRelease(void* user_data) {}
```

Comment:
I'm having trouble following the ownership model for the texture; why doesn't this need to do anything? I would naively expect there to be some memory that needs to be freed.